client/whispers/route.tsx

```diff
@@ -20,5 +20,5 @@ export function WhisperRouteComponent(props: { params: any }) {
     return null
   }
 
```

Comment:
Can you remove the `queueMicrotask` piece above this? (Just call `replace('/')` directly, not inside a microtask)

I had to deal with this in chat too where it caused weird issues, guess I didn't see that whispers was doing it too

Reply:
Should it be removed from here as well?
https://github.com/ShieldBattery/ShieldBattery/blob/7ee9a4fac8bd8a67bbfbe40efa885b3f1f356f55/client/chat/channel-route.tsx#L17-L19